### DISCUSSION TITLE
Is there a way to prioritize alias?

### DISCUSSION THREAD

**Discussion by ooker777** [tadashi-aikawa/obsidian-another-quick-switcher#88] [Q&A] [answered]
It seems that it doesn't so. Say my alias is `"2.01"`. If I search for `201` then it doesn't return like how the default quick switcher can. I have to explicitly have the dot

**Comment by tadashi-aikawa** [comment, marked as answer]
@ooker777 
Does "alias" mean the one in https://help.obsidian.md/How+to/Add+aliases+to+note ?

**Reply by ooker777** [reply]
yes

**Reply by tadashi-aikawa** [reply]
Another Quick Switcher doesn't search fuzzy, so `"201"` doesn't match `"2.01"`.

> It does not search very fuzzy
> (README)

**Reply by ooker777** [reply]
ah I see. Now I understand that part. Is there any reason to not adding that too? Is it because of you not having time?

**Reply by tadashi-aikawa** [reply]
Personally, The reason is I don't need a fuzzy match. And that is also one of the reasons I started developing Another Quick Switcher.

**Reply by ooker777** [reply]
Why isn't it useful to you? Isn't that it helps searching quicker?

**Reply by tadashi-aikawa** [reply]
Please see the comment.

https://forum.obsidian.md/t/remove-strict-word-order-requirement-in-quick-switcher-and-link-suggestions/10966/3

**Reply by tadashi-aikawa** [reply]
And I think it may be possible to fuzzy search by developing that adds options.

**Reply by ooker777** [reply]
I'm confused. As how I understand the thread says Obsidian doesn't support fuzzy search, and AQS is the solution for this lack?

**Reply by tadashi-aikawa** [reply]
- Obsidian supports fuzzy search ("201" matches "2.01")
    - AQS doesn't support it  ("201" doesn't match "2.01")
- Obsidian doesn't support other than strict order ("aa bb" doesn't match "bb aa")
    - AQS supports it  ("aa bb" matches "bb aa")

> Including the reverse order (or any mixed ordering) makes the search results way too noisy (at least with fuzzy search)

That is why I don't have implemented a fuzzy search.

**Comment by ooker777** [comment]
I understand now. Thanks for your patience

**Reply by tadashi-aikawa** [reply]
@ooker777 
It's been a while :)  Now, I support fuzzy search in the following release 🚀 
https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/102#issuecomment-1537365773
